packages/terra-embedded-content-consumer/src/EmbeddedContentConsumer.jsx

```diff
@@ -37,8 +44,26 @@ const propTypes = {
 
 class EmbeddedContentConsumer extends React.Component {
   componentDidMount() {
+    // Merging the iframe options props
+    const frameOptions = { ...this.props.options };
+
+    if (this.props.title) {
```

Comment:
```suggestion
    frameOptions.iframeAttrs = this.props.title && frameOptions.iframeAttrs ? frameOptions.iframeAttrs " {}
```

Reply:
@sdadn and all, **for an iFrame to be accessible it must have the title attribute.** By including "title" as a first class prop we can implement measures to ensure the title is provided and eventually make it required. As is, it is not guaranteed that any engineer will add it. Yes we can beef up our documentation, but that also requires people to read which they do not regularly do. I am specifically asking for this prop to be added to ensure we help enforce best practices. While we have to make it a passive change for now, I will push to make it required as soon as possible.

Reply:
Re-worked the logic with https://github.com/cerner/terra-framework/pull/1776/commits/3ffd8305ee593960880ba6b3ac9bbdab84932e6c